kpops/components/streams_bootstrap_v2/base.py

```diff
@@ -111,16 +112,16 @@ class StreamsBootstrapValues(HelmAppValues):
     )
 
 
-class StreamsBootstrap(HelmApp, ABC):
-    """Base for components with a streams-bootstrap Helm chart.
+class StreamsBootstrapV2(KafkaApp, HelmApp, ABC):
```

Comment:
can we add a deprecation message when using this component?

Reply:
Should we use something like https://github.com/tantale/deprecated

Reply:
yes, that would be great. Additionally, I was thinking we could print a message during instantiation using a Pydantic model validator.

Reply:
Okay there is a deprecated decorator built in https://peps.python.org/pep-0702/

Reply:
@disrupted added both `@depricted()` and `model_validator`

Reply:
is that compatible with our Python version dependency? The PEP says 3.13

Reply:
ah, I see that you import it from typing_extensions 👍🏻